platform/darwin/src/MGLPolyline.mm

```diff
@@ -52,6 +53,20 @@ - (BOOL)isEqual:(id)other {
     return self == other || ([other isKindOfClass:[MGLPolyline class]] && [super isEqual:other]);
 }
 
+- (CLLocationCoordinate2D)coordinate {
+    CLLocationCoordinate2D centroid;
+    
+    mbgl::Polygon<double> multiLineString;
```

Comment:
Polylabel probably isn’t the right tool for this job. In JavaScript, we’d use [turf-line-distance](https://github.com/Turfjs/turf/tree/master/packages/turf-line-distance) to determine the length of the polyline, then use [turf-along](https://github.com/Turfjs/turf/tree/master/packages/turf-along) to get the midway point. It might be interesting to port this functionality to C or C++ for use in the SDKs, but that’s enough work that we should probably tackle it in a separate PR.